camp_fin/merge_candidates.py

```diff
@@ -0,0 +1,130 @@
+"""
+Copyright (c) 2007 Michael Trier
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+Developer's note:
+The get_generic_fields and merge_candidates functions were copied from the
+merge_model_instances management command from django-extensions. The code
+required a small change to work for us, and I didn't want to introduce a large
+dependency for two functions. Permalink to source code:
+https://github.com/django-extensions/django-extensions/blob/b7dfee0be0c16ba7fca107380f81fc86b6aaaaa8/django_extensions/management/commands/merge_model_instances.py
+"""
+import sys
```

Comment:
looks like this "sys" calls are still here? drop those, and this is good to merge.

Reply:
Whoops, removed the calls but forgot the import. Removed!